edn_format/edn_dump.py

```diff
@@ -92,6 +93,8 @@ def udump(obj,
         return '({})'.format(seq(obj, **kwargs))
     elif isinstance(obj, list):
         return '[{}]'.format(seq(obj, **kwargs))
+    elif isinstance(obj, ImmutableList):
+        return '[{}]'.format(seq(obj, **kwargs))
```

Comment:
You could merge this with the `elif` above it:
```python
    elif isinstance(obj, list) or isinstance(obj, ImmutableList):
        return '[{}]'.format(seq(obj, **kwargs))
```

Reply:
@konr @bfontaine Or we can use a tuple:

```python
elif isinstance(obj, (list, ImmutableList)):
        return '[{}]'.format(seq(obj, **kwargs))
```

Reference : https://devdocs.io/python~3.7/library/functions#isinstance

Reply:
Yep. I was wondering if it wasn’t supported in Python 2 but it was added in 2.2 so we’re safe here:

https://devdocs.io/python~2.7/library/functions#isinstance